nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/test/java/org/apache/nifi/integration/accesscontrol/ITProcessorAccessControl.java

```diff
@@ -399,6 +400,43 @@ public void testNoneUserDeleteProcessor() throws Exception {
         verifyDelete(helper.getNoneUser(), NONE_CLIENT_ID, 403);
     }
 
+    /**
+     * Tests attempt to create a restricted processor.
+     *
+     * @throws Exception
```

Comment:
Checkstyle fails on Javadoc with empty message. I changed the line to `* @throws Exception if there is an error creating this processor` to resolve.